kpi/tests/test_permissions.py

```diff
@@ -815,23 +831,16 @@ def test_implied_partial_submission_permission_with_different_filters(self):
                 {'_submitted_by': self.someuser.username},
             ],
             PERM_DELETE_SUBMISSIONS: [
-                {'_submission_date': {'$lte': '2021-01-01'}},
-                {'_submission_date': {'$gte': '2020-01-01'}},
+                {'_submission_date': {'$and': [{'$lte': '2021-01-01', '$gte': '2020-01-01'}]}},
```

Comment:
@noliveleger take note of this. This is the only way we allow end user set $and permissions with the current approach. It supports any mongo helper allowed operation. Implied permissions concerns itself with the top level permissions.